doc/whats_new/v0.24.rst

```diff
@@ -357,6 +357,14 @@ Changelog
   cases support the `binary_only` estimator tag.
   :pr:`17812` by :user:`Bruno Charron <brcharron>`.
 
+:mod:`sklearn.calibration`
+.........................
```

Comment:
```suggestion
..........................
```

This will fix the documentation CI tests, see: 

https://113449-843222-gh.circle-artifacts.com/0/doc/_changed.html

> `doc/whats_new/v0.24.rst:361: WARNING: Title underline too short.`